src/arrow_parquet/uri_utils.rs

```diff
@@ -92,48 +88,40 @@ fn object_store_with_location(uri: &Url, copy_from: bool) -> (Arc<dyn ObjectStor
 async fn get_s3_object_store(bucket_name: &str) -> AmazonS3 {
     let mut aws_s3_builder = AmazonS3Builder::new().with_bucket_name(bucket_name);
 
-    let is_test_running = std::env::var("PG_PARQUET_TEST").is_ok();
-
-    if is_test_running {
-        // use minio for testing
-        aws_s3_builder = aws_s3_builder.with_endpoint("http://localhost:9000");
-        aws_s3_builder = aws_s3_builder.with_allow_http(true);
-    }
+    // AWS_ALLOW_HTTP
+    if let Ok(aws_allow_http) = std::env::var("AWS_ALLOW_HTTP") {
+        aws_s3_builder = aws_s3_builder
+            .with_allow_http(aws_allow_http.parse().unwrap_or_else(|e| panic!("{}", e)));
+    };
 
```

Comment:
probably good to add some context in a comment on why we have this additional logic

at this point, I don't have a very good mental model of what object_store does and does not do for us.

Reply:
Added comment to the function. 

Allowed passing the environment variables to `object_store` as it is. I was not passing them since we do not read all corresponding configs from the config file (at fallback) but now I see no downside passing all the env vars (except some of the configs, that can be read from environment, cannot be read from the config file). (context: object_store can not read from config files)